provider/go.go

```diff
@@ -108,19 +142,41 @@ func (p *Go) BoolGetter() func() (bool, error) {
 }
 
 func (p *Go) paramAndEval(param string, val any) error {
+	if str, ok := val.(string); ok {
+		val = "\"" + str + "\""
+	}
+
 	_, err := p.vm.Eval(fmt.Sprintf("%s := %v;", param, val))
 	if err == nil {
-		_, err = p.vm.Eval(p.script)
+		_, err = p.vm.Eval(fmt.Sprintf("param := %v;", param))
+	}
+	if err == nil {
+		_, err = p.vm.Eval(fmt.Sprintf("val := %v;", val))
+	}
+	if err == nil {
+		var v reflect.Value
+		v, err = p.vm.Eval(p.script)
+		if err == nil && p.result != nil {
+			err = transformSetterGo(p.result, v)
```

Comment:
damit legst du die getter/setter bei jedem durchlauf neu an, im Zweifel wird ein MQTT Broker initialisiert. Das sollte nur 1x passieren?

Reply:
Ich schaue mal, ob ich das in die Initialisierung reinbekomme.

Reply:
Denke das ist dann eine map of provider aus der man wieder den richtigen Typ raussuchen/testen muss. Genau wie bei den `New...FromConfig`. Vllt. lässt sich da noch was wiederverwenden.

Reply:
Wird jetzt beim Parsen am Anfang einmal initalisiert.